Created type error in python errors entry (#875)
Codecademy/docs: content/python/concepts/errors/errors.md

---
Title: 'Errors'

> Description: 'There are two main kinds of errors in Python: syntax errors and exceptions. Exceptions are any errors that arise from syntactically correct code.

Reviewer: ```suggestion
Description: 'There are two main kinds of errors in Python: syntax errors and exceptions. Exceptions are any errors that arise from syntactically correct code.'
```

Noticed this was missing a closing single-quote. This suggestion should fix the one GitHub (test) check that keeps failing. 😄

> Description: 'There are two main kinds of errors in Python: syntax errors and exceptions. Exceptions are any errors that arise from syntactically correct code.'

Reviewer: ```suggestion
Description: 'The two types of errors in Python are syntax errors and exceptions. Exceptions may arise even if the code is syntactically correct.'
```

> Syntax errors, also known as parsing errors, are perhaps the most common kind of complaint you get while you are still learning Python:

Reviewer: ```suggestion
Syntax errors (also known as parsing errors) occur when a sequence of characters, or tokens, violates the syntax of the Python programming language:
```

For the rest of this entry, let's replace all instances of "you" with a neutral, more manual-like tone that doesn't address the reader(s) directly.

Reviewer: Also, on line # 15, can we change to make it look similar to the `Description` on line # 3. Also, since "errors" is the entry concept, let's place it **in bold**. Maybe something like this:

"The two types of **errors** are syntax errors and exceptions."

> The parser repeats the offending line and displays a little arrow `^` pointing at the earliest point in the line where the error was detected.

Reviewer: ```suggestion
The parser repeats the offending line and displays a little caret symbol `^` pointing at the earliest point in the line where the error was detected.
```

> 

Reviewer: Also, on line # 30, can we do the following:

1. Switch around "function" and "print()" with each other. (i.e., ~"... at the function `print()`, since a ..."~ --> "... at the `print()` function, since a ...")
2. Add a hyperlink to "`print()`" by linking out to the following Docs term entry: https://www.codecademy.com/resources/docs/python/built-in-functions/print

ModuleNotFoundError: No module named 'notamodule'
```

### Type Error

`TypeError` is thrown when a function's argument is of an inappropriate type.

> ```error

Reviewer: ```suggestion
```shell
```

Since \```error [doesn't seem to be supported](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/#code-blocks), and this is technically output, can we replace all instances of \```error with \```shell?
